semantics/c/language/translation/init.k

```diff
@@ -44,7 +48,89 @@ module C-TRANSLATION-INIT
           => UnknownCabsLoc
      rule filterCabsLoc(L:CabsLoc) => L [owise]
 
-     rule ExpressionLoc(K:KItem, _) => K
+     rule ExpressionLoc(K:K, L::CabsLoc) => ExpLoc(K, L)
+     rule typeof(ExpLoc(K:K, L::CabsLoc) => K)
+     rule utype(ExpLoc(K:K, L::CabsLoc) => K)
```

Comment:
These rules confuse me. As I understand these functions don't normally apply to expressions?